File: 3019-furthest-point-from-origin/3019-furthest-point-from-origin.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int n = moves.length();
        int left = 0, right = 0, extra = 0;
        for (int i = 0; i < n; i++) {
            char current = moves.charAt(i);
            if (current == 'L') left++;
            else if (current == 'R') right++;
            else extra++;
        }
        return Math.abs(left - right) + extra;
    }
}

File: 3019-furthest-point-from-origin/3019-furthest-point-from-origin.py
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        n = len(moves)
        left, right, extra = 0, 0, 0
        for i in range(n):
            if moves[i] == 'L': left += 1
            elif moves[i] == 'R': right += 1
            else: extra += 1
        
        return abs(left - right) + extra
        